setup.py

```diff
@@ -2117,10 +2116,13 @@ def convert_optional_data_files(files):
             ),
         ),
     ]
-    +
     # And data files convert_data_files can't handle.
-    [
+    + [
+        ("", (str(version_file_path),)),
+        ("pythonwin", (str(scintilla_licence_path),)),
+        ("win32comext/mapi", (str(mapi_stubs_licence_path),)),
         ("win32com", ("com/License.txt",)),
+        ("win32comext", ("com/License.txt",)),
```

Comment:
i don't quite get the first arg here and why it's not specified below (or why win32com already did that :) But why is this even needed given the files added up above?